src/features/post/inFeed/compact/CompactPost.tsx

```diff
@@ -127,11 +136,13 @@ export default function CompactPost({ post, communityMode }: PostProps) {
           </Actions>
         </Aside>
       </Content>
-      <EndDetails>
-        <VoteButton type="up" postId={post.post.id} />
-        <VoteButton type="down" postId={post.post.id} />
-      </EndDetails>
-
+      {compactThumbnailPositionType === "right" && <Thumbnail post={post} />}
+      {compactShowVotingButtons === true && (
```

Comment:
No need for === true

Reply:
Sorry, it was a leftover from before where I was comparing against a typed value.